Makefile

```diff
@@ -159,6 +159,10 @@ ifndef test_spec
 endif
 	@$(VENV) $(TEST_CMD) $(test_spec)
 
+test-logs:
+	mkdir -p servroot/logs && touch servroot/logs/error.log
```

Comment:
`mkdir` and `touch` are not needed.  `tail -F` does not require the file to be present to work.

Reply:
dang... should have tested that. Fixed, thx.